### PR TITLE
CAN-217: Account Settings: Frontend design implementation of supporte…

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
…d camps page

Account Settings: Frontend design implementation of supported camps page
and Formatting

### CODE REVIEW

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                </p>
+                <p>
+                  2. &nbsp;
+                  <span className={styles.Bluecolor}>Digital Identity</span>
+                </p>
+                <p>
+                  3. &nbsp;<span className={styles.Bluecolor}>Prototype</span>
+                </p>
+              </div>
+              <a className={styles.mrgn_left} onClick={showViewMoreModal}>
```
Comment (akuks) [inline src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Probably `Link` should be used here

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+          </Row>
+        </div>
+      </Card>
+      <Card
+        className={styles.cardBox_tags}
+        type="inner"
+        size="default"
+        title={<CardTitle value="Front End Language" />}
+        extra={
+          <a
```
Comment (akuks) [inline src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Not sure if we can use Link here. Can you please check

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+                </p>
+                <p>
+                  2. &nbsp;
+                  <span className={styles.Bluecolor}>Digital Identity</span>
+                </p>
+                <p>
+                  3. &nbsp;<span className={styles.Bluecolor}>Prototype</span>
+                </p>
+              </div>
+              <a className={styles.mrgn_left} onClick={showViewMoreModal}>
```
Comment (akuks) [inline src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Probably `Link` should be used here

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+            </Col>
+            <div></div>
+            <Col span={12}>
+              <div className={styles.line_height1}>
+                <p>
+                  <b>Current Supported Camps:</b>
+                </p>
+                <p>
+                  1. &nbsp;
+                  <span className={styles.Bluecolor}>
```
Comment (akuks) [inline src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: I noticed this section is almost repeated. Can we iterate of create component and pass props?

File: src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+          <a
+            className={styles.RemoveCardSupported}
+            onClick={() => RemoveCardSupportedCamps()}
+          >
+            <CloseCircleOutlined /> Remove support{" "}
+          </a>
+        }
+        style={{ width: 760, marginBottom: 16 }}
+      >
+        {/* <p>1.{" "} <b style={{"color":"#6A90C4"}}> Next.js</b></p>
```
Comment (akuks) [inline src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx]: If not required, we can remove this

File: src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+          <a
+            className={styles.RemoveCardSupported}
+            onClick={() => RemoveCardSupportedCamps()}
+          >
+            <CloseCircleOutlined /> Remove support{" "}
+          </a>
+        }
+        style={{ width: 760, marginBottom: 16 }}
+      >
+        {/* <p>1.{" "}<b style={{"color":"#6A90C4"}}> Next.js</b></p> */}
```
Comment (akuks) [inline src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx]: if not required, we can remove this

**Review by akuks — CHANGES_REQUESTED** [review]
Please go through the review comments

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Done

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Done

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx (hunk 3 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Done

File: src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx (hunk 4 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DelegatedSupportCamps/DelegatedSupportCampsUI/index.tsx]: Done

File: src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx (hunk 5 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx]: Done

File: src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx (hunk 6 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx]: Done

**Review by akuks — APPROVED** [review]

**Pull request merged**